src/System.Drawing.Common/src/System/Drawing/Imaging/Metafile.cs

```diff
@@ -243,8 +243,11 @@ public Metafile(IntPtr referenceHdc, Rectangle frameRect, MetafileFrameUnit fram
         /// </summary>
         public Metafile(string fileName, IntPtr referenceHdc, EmfType type, String description)
         {
-            // Called in order to emulate exception behavior from netfx related to invalid file paths.
-            Path.GetFullPath(fileName);
+            if (fileName == null)
```

Comment:
Hmm, isn't this effectively what `Path.GetFullPath` will do?

The .NET Framework codebase definitely calls `Path.GetFullPath`, it's just done indirectly through `IntSecurity.DemandReadFileIO(fileName);`.

http://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/Advanced/Metafile.cs,358

Reply:
Yes, but `Path.GetFullPath` will set `path` as the argument name, whereas the unit tests expect it to be `null`; see e.g. `Ctor_InvalidPathII_ThrowsArgumentException`.

I guess in this case the .NET Core behavior is an improvement and we could amend the tests to assert that ParamName is `path` on Core?

Reply:
> I guess in this case the .NET Core behavior is an improvement and we could amend the tests to assert that ParamName is path on Core?
 
Yeah, we have some functions in AssertExtensions.cs that allow you to specify a different string name on core / netfx to allow for this. We use it in some other parts of the test.